app/views/users/_form.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_for @user do |f| %>
```

Comment:
Have you tried accessing the edit path after creating an user `users/1/edit`? This is raising an error. Take a look at your `routes.rb` and see if if they're all right.

In addition, also check your routes with `$ rails routes` or more specifically `$rails routes | grep user`. Take note of the fact that there is no `user_path` available and that `@user` is converting to `user_path`.

We need to be able to edit an user.

[`REQUIRED`]

Reply:
Thank you for pointing that out, I will take a look at it.

Reply:
I just fixed the issue and pushed again